src/mod_blocking.erl

```diff
@@ -24,9 +24,11 @@
 
 -type listitem() :: #listitem{}.
 
-start(HostType, _Opts) ->
+-spec start(HostType::mongooseim:host_type(), Opts::gen_mod:module_opts()) -> ok.
```

Comment:
```suggestion
-spec start(HostType :: mongooseim:host_type(), Opts :: gen_mod:module_opts()) -> ok.
```
and personally I would even delete the variable names, I don't think they bring any extra information:
```suggestion
-spec start(mongooseim:host_type(), gen_mod:module_opts()) -> ok.
```